pkg/clients/clients.go

```diff
@@ -78,8 +78,8 @@ type Repository interface {
 	// RetrieveAll retrieves all clients.
 	RetrieveAll(ctx context.Context, pm Page) (ClientsPage, error)
 
-	// RetrieveAllBasicInfo list all clients only with basic information.
-	RetrieveAllBasicInfo(ctx context.Context, pm Page) (ClientsPage, error)
+	// SearchBasicInfo list all clients only with basic information.
+	SearchBasicInfo(ctx context.Context, pm Page) (ClientsPage, error)
```

Comment:
I think this change is not required, since we have SearchBasicInfo function in things postgres